web/src/css/6-components/_know-your-rights-route.scss

```diff
@@ -0,0 +1,21 @@
+.RightsList {
+    font-family: Times;
```

Comment:
Nit picks here:

- General preference for alphabetizing properties. When these things get larger, it's much easier to find a property you're looking for if there's an order to it, and the only order that's ever understandable is alphabetical order.
- I'd just let the standard `font-family` get inherited, so you can remove this line.
- `align-items` isn't needed anymore
- `justify-content` isn't needed anymore
- I don't think you need position `absolute`